root/report/AnnotationsArtists.js

```diff
@@ -0,0 +1,46 @@
+/*
+ * @flow
+ * Copyright (C) 2018 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+
+import {withCatalystContext} from '../context';
+import Layout from '../layout';
+import formatUserDate from '../utility/formatUserDate';
+import {l} from '../static/scripts/common/i18n';
+
+import ArtistAnnotationList from './components/ArtistAnnotationList';
+import FilterLink from './FilterLink';
+import type {ReportArtistAnnotationT, ReportDataT} from './types';
+
+const AnnotationsArtists = ({
+  $c,
+  canBeFiltered,
+  filtered,
+  generated,
+  items,
+  pager,
+}: ReportDataT<ReportArtistAnnotationT>) => (
+  <Layout fullWidth title={l('Artist annotations')}>
+    <h1>{l('Artist annotations')}</h1>
+
+    <ul>
+      <li>{l('This report lists artists with annotations.')}
+      </li>
+      <li>{l('Total artists found: {count}', {count: pager.total_entries})}</li>
+      <li>{l('Generated on {date}', {date: formatUserDate($c.user, generated)})}</li>
```

Comment:
There's something strange going on with the date display. I generated some reports, and the master branch displays the date as "2019-01-19 03:52 UTC" (which is correct), while this branch displays it as "2019-01-18 06:00 UTC" (very strange since the minutes are wrong too).

Reply:
Fixed the specific issue here, plus created https://tickets.metabrainz.org/browse/MBS-9990 for the weird date issue.